src/makim/cli/auto_generator.py

```diff
@@ -286,3 +286,40 @@ def create_dynamic_command_cron(
 
     exec(function_code, global_vars, local_vars)
     decorator(local_vars['dynamic_command'])
+
```

Comment:
I guess this implementation could be moved to cli/pipelines.py?
as a reference, maybe it is similar to this? https://github.com/osl-incubator/makim/pull/126/files#diff-62bbe6d69c35ea6721375139f71770ef87e671c77b209f1beb151ffff0abb63fR25